### DISCUSSION TITLE
Feature: If `packages` are in the Standard Library, ignore them

### DISCUSSION THREAD

**Discussion by JeffersGlass** [pyscript/pyscript#1824] [Proposals]
Just a small quality-of-life feature proposal for the future - if a user includes a package in the py-config `packages` key that is from the standard library, just ignore it and maybe emit a warning.

For example, this currently errors with a (semi-cryptic) warning about a lack of Pure Python packages:

```html
<py-config>
    packages = ['enum']
</py-config>
```

The proposal is that this would instead emit a warning like `'enum' is already in the Standard Library and will not be installed`. Or something. 

This might get more complicated as we consider `py-config` vs `mpy-config`, the ability to include different interpreters with the `interpreters` key, etc. Haven't thought through all that, it's just a user error that's come up a few times over the past year that would be nice if we can protect them from.

**Comment by WebReflection** [comment]
This looks like a request for Pyodide ... we just pass the packages list AS-IS to pyodide, anything else screams for slower bootstraps to me

**Comment by bugzpodder** [comment]
you can also try pkgutil.iter_modules or importlib.util.find_spec
